packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -55,27 +54,38 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
     private divRef: any;
     private yAxisRef: any;
     private mouseIsDown = false;
+    private isPanning = false;
+    private isSelecting = false;
     private positionXMove = 0;
     private positionYMove = 0;
     private isRightClick = false;
     private isMouseMove = false;
     private posPixelSelect = 0;
     private isMouseLeave = false;
+    private mousePanningStart = BigInt(0);
+    private resolution = 0;
     private startPositionMouseRightClick = BigInt(0);
     private margin = { top: 15, right: 0, bottom: 5, left: this.getYAxisWidth() };
     private isScatterPlot: boolean = this.props.outputDescriptor.id.includes('scatter');
     private plugin = {
         afterDraw: (chartInstance: Chart, _easing: Chart.Easing, _options?: any) => { this.afterChartDraw(chartInstance); }
     };
 
-    private endSelection = () => {
+    private endSelection = (event: any) => {
         if (this.isRightClick) {
            this.applySelectionZoom();
         }
         this.mouseIsDown = false;
-        if (!this.state.shiftKey) {
-            this.setState({ cursor: 'default'});
+        this.isSelecting = false;
+        this.isPanning = false;
+        if (!event.shiftKey && !event.ctrlKey) {
+            this.setState({ cursor: 'default' });
+        } else if (event.ctrlKey) {
```

Comment:
Both keys pressed leads to grabbing, should be default. I suggest:
if (shift !ctrl) crosshair else if (ctrl !shift) grabbing else default.